components/Downloads/ChangelogModal/index.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Button from '@/components/Common/Button';
+import { githubProfileAvatarUrl } from '@/util/gitHubUtils';
+
+import ChangelogModal from '.';
```

Comment:
```suggestion
import ChangelogModal from '@/components/Downloads/ChangelodgModal';
```

We always use absolute imports on TS/TSX files